src/Background/estimators.jl

```diff
@@ -61,3 +61,30 @@ julia> estimate_background(Mode, data)
 struct Mode <: BackgroundEstimator end
 
 estimate_background(::Mode, data; dims = :) = mode(data)
+
+
+"""
+    MMMBackground(median_factor=3, mean_factor=2)
+
+This estimator is originally taken from DAOPHOT.
+The background is estimated as `median_factor*median - mean_factor*mean`, the default values of `median_factor` and `mean_factor` are `3` anf `2` respectively.
```

Comment:
Generally the first sentence of the doc string should be pretty descriptive. "this is taken from DAOPHOT" is not quite the description I think we want. I think a better way to go about it may be 
```julia
"""
    MMMBackground(median_factor=3, mean_factor=2)

Estimate the background using a mode estimator of the form `median_factor * median - mean_factor * mean`. 
d
This algorithm is based on the `MMM` routine originally implemented in DAOPHOT. `MMM` uses factors of `median_factor=3` and `mean_factor=2` by default. This estimator assumes that contaminated sky pixel values overwhelmingly display positive departures from the true value.

# Examples
[...]

# See Also
[`SourceExtractor`](@ref)
"""
```